tests/playwright/src/model/pages/volumes-page.ts

```diff
@@ -39,47 +39,55 @@ export class VolumesPage extends MainPage {
   }
 
   async openCreateVolumePage(volumeName: string): Promise<CreateVolumePage> {
-    const row = await this.getVolumeRowByName(volumeName);
-    if (row !== undefined) {
-      throw Error('Volume is already created');
-    }
-
-    await playExpect(this.createVolumeButton).toBeEnabled();
-    await this.createVolumeButton.click();
-    return new CreateVolumePage(this.page);
+    return await test.step('Open Create Volume Page', async () => {
```

Comment:
same remark, usually we return the promise, not the await of a promise

Reply:
Same as above, the await is for test.step, the base functionality has not changed.